### DISCUSSION TITLE
Yes, you translated part of it, but kept part of it

### DISCUSSION THREAD

**Discussion by Rain-0x01-39** [ppy/osu#28498] [Q&A] [answered]
<details><summary>...</summary>
<p>

<del>我去除了大肠的大部分腥味，但是保留了一部分</del>

</p>
</details> 

![image](https://github.com/ppy/osu/assets/83620631/ede7693d-3ac8-401d-a429-cb19e665eaf3 "so")
![osu_2024-06-16_19-50-50](https://github.com/ppy/osu/assets/83620631/e5de46b8-1064-47a1-a58b-303fbfb6fda7)
![osu_2024-06-16_19-56-57](https://github.com/ppy/osu/assets/83620631/d8434192-9014-4071-8ddd-bc84fd36d05b)
![osu_2024-06-16_19-58-10](https://github.com/ppy/osu/assets/83620631/6eacb1ce-9f62-4f21-99ec-e748fc1f0453)
![osu_2024-06-16_19-59-38](https://github.com/ppy/osu/assets/83620631/86f4c0ff-b375-4a9e-8228-35ee49f5cb02)

There may be other places where there is no translation.

My language is Chinese Simplified

**Comment by Rain-0x01-39** [comment]
Other languages may have this problem as well.

**Comment by gi-b716** [comment]
我也是简体中文玩家，我遇到了同样的问题
事实上，我在crowdin和osu-resources都看了一遍，没有找到未翻译的.resx，所以我猜有一些本地化文件根本没有创建
:(

说到这个，我认为不仅是游戏内文本需要翻译，内嵌的osu!wiki语言切换也该继续推进 #23296 

---

I am also a simplified Chinese player, and I have encountered the same problem
In fact, I looked in both crowdin and osu-resources and found no untranslated .resx, so I guess some localization files are not created at all
:(

Speaking of which, I think not only the in-game text needs to be translated, but also the built-in osu!wiki language switch should be promoted #23296

**Reply by gi-b716** [reply]
PTAL

**Comment by bdach** [comment, marked as answer]
Yes we know. Some things are more difficult to translate than others.

**Comment by Rain-0x01-39** [comment]
有一些本地化文件**根本没有创建**。这导致社区无法翻译这部分文字。

--------
There are some localization files that are **not created at all**. This resulted in the community not being able to translate this part of the text.

**Comment by Rain-0x01-39** [comment]
even more outrageous
![osu_2024-06-19_15-32-43](https://github.com/ppy/osu/assets/83620631/f9bd0ba8-baff-410a-9e23-49a69a194832)
